Dockerfile

```diff
@@ -14,8 +14,7 @@ RUN CGO_ENABLED=0 GOOS=linux go build -o /bin/exporters .
 EXPOSE 8090
 
 
-FROM registry.access.redhat.com/ubi9/ubi-minimal:9.3-1361.1699548032
-RUN microdnf update --setopt=install_weak_deps=0 -y && microdnf install -y libcurl-minimal libcurl-devel
+FROM registry.access.redhat.com/ubi9-micro@sha256:8e33df2832f039b4b1adc53efd783f9404449994b46ae321ee4a0bf4499d5c42
```

Comment:
This seems unrelated to the fix but a welcome change regardless. I just want to understand the justification. The rest LGTM.

Reply:
Integration test is failing due to ubi-minimal image (I think some recent changes in build) so updated the image to ubi-micro